__tests__/app.test.js

```diff
@@ -39,10 +39,44 @@ describe("/api", () => {
         .get('/api')
         .expect(200)
         .then((response) => {
-            console.log(response.body)
+            //console.log(response.body)
```

Comment:
Please delete console.logs before pushing code. Commenting out isn't sufficient, this line should be deleted